### PR TITLE
KPIV-17: Adding wait-mod to member-flags when build conference profile

### PR DESCRIPTION
**Pull request by dhuynh-Kazoo**
Add `wait-mod` to `member-flags` of the conference profile. The reason is to wait until the participant joins with moderator flag in dial conference command to start the conference. That makes kazoo conference compatible with Twilio Conference `startConferenceOnEnter` attribute

### CODE REVIEW

**Comment by dhuynh-Kazoo** [comment]
Linked PR: https://github.com/2600hz/kazoo-ecallmgr/pull/260

File: src/conf_config_req.erl [hunk 1]
```diff
@@ -340,6 +340,7 @@ member_flags(Conference) ->
     Flags = [{kapps_conference:member_join_muted(Conference), <<"mute">>}
             ,{kapps_conference:member_join_deaf(Conference), <<"deaf">>}
             ,{kapps_conference:participant_join_video_muted(Conference), <<"vmute">>}
+            ,{kapps_conference:wait_for_moderator(Conference), <<"wait-mod">>}
```
Comment (icehess) [inline src/conf_config_req.erl:343]: As we discuss this, `wait-mod` is **NOT** a member-flags it is a `conference-flags`:
```
<action application="set" data="conference_flags=wait-mod,audio-always"/>
```

**Review by icehess — CHANGES_REQUESTED** [review]

File: src/conf_config_req.erl (hunk 1 above)
Comment (dhuynh-Kazoo) [reply src/conf_config_req.erl:343]: Oh sorry I misunderstood from the doc, fixing it now

Comment (dhuynh-Kazoo) [reply src/conf_config_req.erl:343]: but it should be `|` not `,`
@icehess  Reference doc: 

```

conference-flags | Can be any combination of allowed values separated by "\|" (pipe character). | wait-mod\|audio-always

```

Comment (icehess) [reply src/conf_config_req.erl:343]: @dhuynh-Kazoo https://github.com/2600hz/kazoo-conference/blob/master/src/conf_config_req.erl#L218-L249

Comment (dhuynh-Kazoo) [reply src/conf_config_req.erl:343]: okay I am tessting it again

**Pull request closed without merging**